admin/services/setting.service.js

```diff
@@ -24,18 +24,22 @@ settingService.update = async (loadingTimeLimit, questionTimeLimit, checkTimeLim
 
 /**
  * Get check settings
+ * @param {Boolean} cacheBust
  * @returns {questionTimeLimit: number, loadingTimeLimit: number, checkTimeLimit: number}
  */
-settingService.get = async () => {
-  let settings = await settingDataService.sqlFindOne()
-  if (!settings) {
-    settings = {
+let cachedSettings
+settingService.get = async (cacheBust = false) => {
```

Comment:
Should there be an expiry on the local cache here? Say 5 minutes?  If not, the settings are unchangeable without a web app reboot.

Reply:
good catch 🥇

Reply:
yea I did miss that